api/src/main/java/org/pmiops/workbench/cdr/model/DbCriteria.java

```diff
@@ -24,6 +24,8 @@
   private boolean group;
   private boolean selectable;
   private String count;
+  private String parentCount;
+  private String childCount;
```

Comment:
We could change these to be Long instead of String to match the database. I think we change  count from String to Long in the db somewhere along the way and we didn't update it here.